lib/xmlrpc/config.rb

```diff
@@ -33,6 +33,29 @@ module Config
     # enable Introspection extension by default
     ENABLE_INTROSPECTION = false
 
+    [
+      :DEFAULT_WRITER,
+      :DEFAULT_PARSER,
+      :ENABLE_NIL_CREATE,
+      :ENABLE_NIL_PARSER,
+      :ENABLE_BIGINT,
+      :ENABLE_MARSHALLING,
+      :ENABLE_MULTICALL,
+      :ENABLE_INTROSPECTION
+    ].each do |option|
+      getter = option.to_s.downcase
+      getter.concat('?') if [true, false].include?(const_get(option))
+      define_singleton_method(getter) do
+        const_get(option)
+      end
+
+      define_singleton_method("#{option.downcase}=") do |value|
+        module_eval do
```

Comment:
Do we need this `module_eval`?

Reply:
Turns out we don't. I copy-pasted this from some code we had for changing the config options, but that was run from another module than `Config`.

I'ts gone now.